test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -102,14 +102,20 @@ void checkBlocks(TestBlock const& _blockFromFields, TestBlock const& _blockFromR
 
 void doBlockchainTests(json_spirit::mValue& _v, bool _fillin)
 {
-	if (!Options::get().fillchain) //fill blockchain through state tests
+	bool skipTestOutput = false;
+	string casename = boost::unit_test::framework::current_test_case().p_name;
+	if (casename == "bcRandom" || Options::get().fillchain)
+		skipTestOutput = true;
+
+	if (!skipTestOutput)
```

Comment:
This variable seems to have reverted meaning. Shouldn't it be when `skipTestOutput == true`, don't do `TestOutputHelper::initTest` ?